.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,55 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug"]
+
+body:
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+      - label: I have searched the existing issues
+        required: true
+  - type: textarea
+    attributes:
+      label: Current Behavior
+      description: A concise description of what you're experiencing.
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Expected Behavior
+      description: A concise description of what you expected to happen.
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Steps To Reproduce
+      description: Steps to reproduce the behavior.
+      placeholder: |
+        1. In this environment...
+        2. With these relevant config options...
+        3. Run '...'
+        4. See error...
+  - type: dropdown
+    id: version
+    attributes:
+      label: Adapter Version
+      description: What version of our software are you running? 
```

Comment:
Maybe "what version of dbt-dremio"?

Reply:
Also should we have one for dremio version as well?

Reply:
Good suggestions. Will add them in.